include/seqan3/core/metafunction/iterator.hpp

```diff
@@ -113,6 +113,110 @@ struct size_type<it_t>
 
 // see specialisation for ranges in core/metafunction/range.hpp
 
+// ----------------------------------------------------------------------------
+// iterator_tag
+// ----------------------------------------------------------------------------
+
+/*!\brief Type metafunction that deduces the
+ * [iterator_category](https://en.cppreference.com/w/cpp/iterator/iterator_tags) from the modelled
+ * concept [Type metafunction].
+ * \tparam it_t The type to operate on.
+ *
+ * <table>
```

Comment:
HTML table because I wanted to reference the concepts (no autolink 😢 ) and the OutputIterator exceeded the line length. Since no linebreaks are allowed in markdown tables, I made a HTML table.

On the plus side, the HTML table looks much nicer than the markdown table...

Reply:
It's ok to exceed line length for markdown tables...